lib/pbench/test/unit/server/auth/test_auth.py

```diff
@@ -630,14 +630,13 @@ def tio_exc(token: str) -> JSON:
         with app.app_context():
             monkeypatch.setattr(oidc_client, "token_introspect", tio_exc)
 
-        """Verify success path of verify_auth_internal"""
-        app = Flask("test-verify-auth-internal")
+        """Verify success path of verify_auth_api_key"""
+        app = Flask("test-verify-auth")
```

Comment:
Line 633 looks like it should just be removed (we already have a better docstring at line 615).

The function argument in line 634 seems like it should be `test-verify-auth-api-key` to match the function name at line 612.  (And, line 628 should be removed, along with most if not all of the code in lines 616-632.)